tests/test_cif_io.py

```diff
@@ -177,4 +177,21 @@ def mockreturn(*args, **kwargs):
         assert actual == expected
         # assert True
 
-# End of file.
+
+mpcontribs_tst = [
+    ('mpc', 'This function is not yet implemented'),
+    ('fs', None),
+    (False, 'mpc_client must be either mpc or fs')
```

Comment:
this only tests if `False` is passed in, which is great but a rather specific case so is not adding much in the way of covering different situations.  

Better coverage is if you pass in a string that is not allowed.  It could be anything, such as `'shiny object'` but to make the tests more readable I would make it `'unknown_string'` or something like that.

Another bad input might be something that is not a string at all, but again you get the most coverage by passing in a general non-string object, but again make the intent visible to the code reader.   There are many ways to do that, but maybe passing in `['non_string_input']`

Hopefully this is helping you to get the idea.